app-backend/datamodel/src/main/scala/Scope.scala

```diff
@@ -182,6 +182,9 @@ object Scope {
       case Scopes.ProjectsFullAccess => Json.fromString("projects:fullAccess")
       case Scopes.RasterFoundryOrganizationAdmin =>
         Json.fromString("organizations:admin")
+      case Scopes.RasterFoundryUser =>
+        Json.fromString("users:member")
```

Comment:
Can we switch this to something more descriptive? `member` doesn't tell us a lot (member of what?), something like `users:platformUser` I think would be clearer